.github/workflows/build_and_test.yml

```diff
@@ -12,8 +12,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.11"]
```

Comment:
I decided to test 3.8 & 3.11 only to reduce the cost on OPENAI. @xinrong-meng any suggestions on this one?